api/openapi/api.yaml

```diff
@@ -167,15 +167,22 @@ paths:
             nullable: true
           description: Dataset SynID
           required: true
+        - in: query
+          name: input_token
+          schema:
+            type: string
+            nullable: false
+          description: Token
+          required: true
       operationId: api.routes.submit_manifest_route
       responses:
         "200":
-          description: Manifest submitted
+          description: Manifest submitted. Synapse ID returned. 
```

Comment:
Instead of Synapse ID, could we use "Manifest ID (e.g. Synapse ID if your asset management platform is Synapse)".